scripts/bin/infra

```diff
@@ -1,4 +1,10 @@
 #!/bin/bash
+
+# Check for the flag SLANG_INFRA_RELEASE before calling `set -u`
```

Comment:
Instead of having this special exception before calling `set -u`, I suggest using default substitution, to replace a non-existing variable with an empty string, and then checking if it has zero length or not:

```bash
if [[ -z "${CI:-}" ]]; then
  # debug
else
  # release
fi
```